api/proto/teleport/legacy/types/events/events.proto

```diff
@@ -3053,6 +3053,48 @@ message Unknown {
   string Data = 4 [(gogoproto.jsontag) = "data"];
 }
 
+// OSType is the same as teleport.devicetrust.v1.OSType.
+// Duplicated because gogo doesn't play well with protoc-gen-go.
+enum OSType {
+  OS_TYPE_UNSPECIFIED = 0;
+  // Linux.
+  OS_TYPE_LINUX = 1;
+  // macOS.
+  OS_TYPE_MACOS = 2;
+  // Windows.
+  OS_TYPE_WINDOWS = 3;
+}
+
+// DeviceTrustMetadata groups device information for events.
+message DeviceTrustMetadata {
```

Comment:
I'm not sure I follow the message renames, is there some convention that I broke?

Reply:
nope, i'll change it back. before the move, `DeviceMetadata` was defined in `devicetrust.proto` making it clear it's for device trusts, and since i moved it out, I thought i needed to change the name with it

Reply:
are you okay with renaming `DeviceEvent` to `DeviceTrust`? that one was breaking the naming convention where in the `events.proto` there is no event name that ends with  `xxxEvent`

Reply:
I'm not sure the rename is worth it at this point. We'll break the builds if we do it this way, plus imo DeviceEvent is a better name for it than DeviceTrust.